client/scss/media.scss

```diff
@@ -10,29 +10,40 @@
 		&>a {
 			padding: 0px;
 			justify-content: center;
-			
+
 			h1 {
 				width: auto;
+				font-size: 12vw;
 			}
 
 			&:after {
 				display: none;
 			}
 		}
-		
+
 		nav {
 			padding: 0px;
 			justify-content: center;
 			border-bottom-color: $color-text-main;
 			width: 100%;
 			margin-left: 0px;
+
+			a {
+				width: 100px;
```

Comment:
It's arbitrary to add more spacing between the header links ("home", "about", "contact") on mobile.  `em` might be a better candidate, but definitely not `%` or `vw`.